cmd/store/delete.go

```diff
@@ -34,6 +35,25 @@ var deleteCmd = &cobra.Command{
 	Example: "fga store delete --store-id=01H0H015178Y2V4CX10C2KGHF4",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		clientConfig := cmdutils.GetClientConfig(cmd)
+		// First, confirm whether this is intended
+		force, err := cmd.Flags().GetBool("force")
+		if err != nil {
+			return fmt.Errorf("failed to obtain force flag %w", err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to parse force flag due to %w", err)
```

just to be consistent with errors elsewhere